tools/cli/go-whisk/whisk/client.go

```diff
@@ -375,14 +375,11 @@ func parseApplicationError(resp *http.Response, data []byte, v interface{}) (*ht
     appErrResult := &AppErrorResult{}
     err = json.Unmarshal(data, appErrResult)
 
-    // Handle application errors that occur with blocking invocations when result is true (#5)
+    // Handle application errors that occur with blocking invocations when --result option is true (#5)
     if err == nil && appErrResult.Error != nil {
         Debug(DbgInfo, "Error code is null, blocking with result invocation error has occured\n")
-        errMsg := wski18n.T(
-            "The following application error was received: {{.err}}",
-            map[string]interface{}{
-                "err": *appErrResult.Error,
-            })
+        errMsg := fmt.Sprintf("%v", *appErrResult.Error)
```

Comment:
yep (when blocking/result options were set on the invocation).  the only change was to remove the `The following application error was received` preamble which cluttered up the output.